lntest/itest/lnd_signer_test.go

```diff
@@ -411,3 +412,75 @@ func deriveCustomizedKey(ctx context.Context, node *lntest.HarnessNode,
 	}
 	return pub, nil
 }
+
+// testSignVerifyMessage makes sure that the SignMessage RPC can be used with
+// all custom ways of specifying by verifying with VerifyMessage. Tests both
+// ecd and schnorr signatures.
+func testSignVerifyMessage(net *lntest.NetworkHarness, t *harnessTest) {
+	runSignVerifyMessage(t, net.Alice)
+}
+
+// runSignVerifyMessage makes sure that the SignMessage RPC can be used with
+// all custom ways of specifying by verifying with VerifyMessage. Tests both
+// ecd and schnorr signatures.
+func runSignVerifyMessage(t *harnessTest, alice *lntest.HarnessNode) {
+	ctxb := context.Background()
+	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
+	defer cancel()
+
+	aliceMsg := []byte("alice msg")
+	keyLoc := signrpc.KeyLocator{KeyFamily: 0, KeyIndex: 1}
```

Comment:
We should use a non-zero key family here since that has some special cases in some places.